scripts/xtensa-build-zephyr.py

```diff
@@ -87,6 +87,23 @@ class PlatformConfig:
 	aliases: list = dataclasses.field(default_factory=list)
 	ipc4: bool = False
 
+# These cannot be built by everyone out of the box yet.
```

Comment:
Eventually toolchain will be availalbe later? Then FIXME or TODO placeholder may be nicer?
Otherwise looks good.

Reply:
> Eventually toolchain will be availalbe later? 

Yes hopefully. As @andyross said: this is a Zephyr SDK issue.